xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3613,9 +3613,9 @@ wysiwyg.admin.colorsPerRow.hint=The number of colors to display per row in the c
 wysiwyg.admin.colorPalette.hint=The colors available in the color picker. You can change any color by clicking on it.
 wysiwyg.admin.font=Font settings
 wysiwyg.admin.fontNames.hint=The list of font names available in the font picker. You can add new font names or remove existing ones.
-wysiwyg.admin.fontNames.add.hint=Add font name..
+wysiwyg.admin.fontNames.add.hint=Add font name...
```

Comment:
An ellipsis is `…`, not `...`, i.e., not three dots but one character. Please also correct this for the other changed messages.

Reply:
I was about to use ```…``` but I saw 2 dots everywhere and thought why not just add a dot.
Sure I'll fix it :)